src/GitHub.VisualStudio/UI/Views/PullRequestListView.xaml.cs

```diff
@@ -59,8 +59,10 @@ public PullRequestListView(IGitHubServiceProvider serviceProvider)
 
         bool disposed;
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA1816:CallGCSuppressFinalizeCorrectly")]
-        public void Dispose()
+        protected override void Dispose(bool disposing)
         {
+            base.Dispose(disposing);
+
             if (disposed) return;
```

Comment:
For types with finalizers, you only want to access managed data if you're inside the `disposing` block. Not that it will happen here, but with finalizers in play, there is a (very very very) small chance that managed instance data is no longer valid when they're running, so accessing instance fields could crash (it only really matters if you're disposing managed and native data, really). `disposing` is the flag that tells you whether it's safe to access managed data during this process, so the `if (disposed)` check should be inside that block (and since `disposed` is only set inside that block, doing the check here or there doesn't change things in practice - it's just the safest pattern for when you're disposing managed and native data and there's finalizers in play)